Content.Server/NPC/HTN/HTNSystem.cs

```diff
@@ -133,6 +133,31 @@ private void OnHTNShutdown(EntityUid uid, HTNComponent component, ComponentShutd
         component.PlanningJob = null;
     }
 
+    public void SetHTNEnabled(Entity<HTNComponent> ent, bool state, float planCooldown = 0f)
```

Comment:
Rider is also complaining about the naming rules here, but `SetHtnEnabled` does not read well.
```suggestion
	// ReSharper disable once InconsistentNaming
    public void SetHTNEnabled(Entity<HTNComponent> ent, bool state, float planCooldown = 0f)
```